op-batcher/txmgr.go

```diff
@@ -49,17 +56,52 @@ func NewTransactionManager(log log.Logger, txMgrConfg txmgr.Config, batchInboxAd
 // This is a blocking method. It should not be called concurrently.
 // TODO: where to put concurrent transaction handling logic.
 func (t *TransactionManager) SendTransaction(ctx context.Context, data []byte) (*types.Receipt, error) {
-	tx, err := t.CraftTx(ctx, data)
+
+	var nid [8]byte
+
+	// FIXME: params
+	copy(nid[:], "e8e5f679bf7116cb")
+
+	res, err := t.daClient.SubmitPFD(ctx, nid, data, 20000, 700000)
+	if err != nil {
+		t.log.Warn("unable to publish tx to celestia", "err", err)
+		return nil, err
+	}
+
+	height := res.Height
+
+	// FIXME: needs to be tx index / share index?
+	index := res.Logs[0].MsgIndex
+
+	// DA pointer serialization format
+	// | -------------------------|
+	// | 8 bytes       | 4 bytes  |
+	// | block height | tx index  |
```

Comment:
I don't think it is safe to rely on this index and don't think the used index provides the information hoping for.

If we want to attach multiple PFD per namespace per block, could add metadata to the stored data bytes for that. Also see note further down about signing.